lib/extensions/break_apart.py

```diff
@@ -0,0 +1,67 @@
+from copy import deepcopy
+
+from shapely import geometry as shgeo
+
+import inkex
+
+from ..elements import AutoFill, Fill
+from ..i18n import _
+from ..svg import get_correction_transform
+from .base import InkstitchExtension
+
+
+class BreakApart(InkstitchExtension):
+    def effect(self):  # noqa: C901
+        if not self.get_elements():
+            return
+
+        if not self.selected:
+            inkex.errormsg(_("Please select one or more fill areas to break apart."))
+            return
+
+        for element in self.elements:
+            if not isinstance(element, AutoFill) and not isinstance(element, Fill):
+                continue
+            if len(element.paths) <= 1:
+                continue
+
+            polygons = []
+            multipolygons = []
+            holes = []
+
+            # sort paths by size and convert to polygons
+            element.paths.sort(key=lambda point_list: shgeo.Polygon(point_list).area, reverse=True)
+            for path in element.paths:
+                polygons.append(shgeo.Polygon(path))
```

Comment:
How about sort `polygons` after the loop?  That way we don't have to convert each path to a `shgeo.Polygon` twice.